sdk/keyvault/keyvault-keys/test/public/keyClient.hsm.spec.ts

```diff
@@ -71,5 +72,82 @@ onVersions({ minVer: "7.2" }).describe(
         );
       });
     });
+
+    onVersions({ minVer: "7.3-preview" }).describe("releaseKey", () => {
```

Comment:
To explain this test setup here:

with the service team's help I setup a mock service on Azure App Service that pretends to be an attestation service and returns mock data. 

The managed HSM requires an attestation service that supports OIDC [OpenID Connect Discovery](https://swagger.io/docs/specification/authentication/openid-connect-discovery/). The minimal requirements for us are:
- It should be reachable by the MHSM
- It should respond to `https://server.com/.well-known/openid-configuration` with the jwks_uri 
- the jwks_uri value should be reachable by the MHSM
- The attestation we provided must have the right information

The last bit is where it gets interesting - we need to use the same keys to sign the JWT as the ones returned by jwks_uri. 

So this should be something that is stood up by live test resources if we want to test this live. You can see what my mock service returns here:

https://malegerskr4.azurewebsites.net/generate-test-token -> used by us to get an attestation signed JWT
https://malegerskr4.azurewebsites.net/keys -> used by the MHSM to get the key used for signing the token
https://malegerskr4.azurewebsites.net/.well-known/openid-configuration -> tells the MHSM where to go to get key

I'll investigate ways to make this part of our test resource deployment, but it's also possible to use this today to record tests for any language